src/utils/triton.h

```diff
@@ -25,9 +25,18 @@
 
 #include "src/dali_executor/io_descriptor.h"
 #include "src/dali_executor/utils/dali.h"
+#include "src/dali_executor/utils/utils.h"
 #include "triton/backend/backend_model.h"
 #include "triton/backend/backend_model_instance.h"
 
+#define TRITON_CALL(expr)     \
+  do {                        \
+    auto err = expr;          \
```

Comment:
```suggestion
    auto err = (expr);        \
```
In case `expr` contains commas.

Reply:
fixed